doc/sphinx-guides/source/api/native-api.rst

```diff
@@ -118,11 +118,18 @@ The fully expanded example above (without environment variables) looks like this
 
 You should expect an HTTP 200 response and JSON beginning with "status":"OK" followed by a representation of the updated Dataverse collection.
 
-Same as in :ref:`create-dataverse-api`, the request JSON supports an optional ``metadataBlocks`` object, with the following supported sub-objects:
+Same as in :ref:`create-dataverse-api`, the request JSON supports a ``metadataBlocks`` object, with the following supported sub-objects:
```

Comment:
Good point, thanks for noticing because it wasn’t properly explained. I’ve mentioned again that it’s optional and added a note explaining what happens if `metadataBlocks` is not sent in the JSON.